gui/src/main.rs

```diff
@@ -147,8 +145,13 @@ fn run_vmm(args: &Args) -> Result<(), ApplicationError> {
 
         todo!()
     } else {
-        match run_launcher(&graphics, profiles)? {
-            Some(v) => v,
+        let vmm_args = run_launcher(&graphics, profiles)?.then_some(VmmArgs {
+            kernel_path,
+            debugger: None,
```

Comment:
How we will construct this field here? I mean when the user click on the start debug button next to the debug address.